source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -984,6 +1065,159 @@ TEST_F(NiSyncDriverApiTest, DISABLED_SetTimeReference8021AS_ReturnsSuccess)
   EXPECT_EQ(VI_SUCCESS, viStatus);
 }
 
+TEST_F(NiSyncDriverApiTest, DISABLED_CreateClearFutureTimeEvent_ReturnsSuccess)
+{
+  ViStatus viStatus;
+
```

Comment:
Can you re-arrange the line breaks to match the test names for this and the other new tests? It might also make sense for tests like this to have separate pairs of `grpcStatusCreate, viStatusCreate` and `grpcStatusClear, viStatusClear` variables so you can call the APIs back to back and have all the Expectations together at the end of the test method.